tests/unit/test_unused_disk.py

```diff
@@ -1,72 +1,91 @@
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
 import pytest
 import find_unused_disk
 import os
 
 
-blkid_data_pttype = [('/dev/sdx', '/dev/sdx: PTTYPE=\"dos\"'),
-                     ('/dev/sdy', '/dev/sdy: PTTYPE=\"test\"')]
+blkid_data_pttype = [
+    ("/dev/sdx", '/dev/sdx: PTTYPE="dos"'),
+    ("/dev/sdy", '/dev/sdy: PTTYPE="test"'),
+]
 
-blkid_data = [('/dev/sdx', 'UUID=\"hello-1234-56789\" TYPE=\"crypto_LUKS\"'),
-              ('/dev/sdy', 'UUID=\"this-1s-a-t3st-f0r-ansible\" VERSION=\"LVM2 001\" TYPE=\"LVM2_member\" USAGE=\"raid\"'),
-              ('/dev/sdz', 'LABEL=\"/data\" UUID=\"a12bcdef-345g-67h8-90i1-234j56789k10\" VERSION=\"1.0\" TYPE=\"ext4\" USAGE=\"filesystem\"')]
+blkid_data = [
```

Comment:
How this can be considered an improvement is beyond my comprehension, but I suppose I can read it either way.

Reply:
python `black` - this is a double edged sword - it fixes many legitimate issues found by Galaxy importer, but it imposes a rather arbitrary, and in some cases hard-to-read, formatting regime

Reply:
Every other system role uses python `black` formatting - if storage doesn't want to use this, we'll have to figure out some way to make an exception, and figure out how to lint some of the issues found by Galaxy importer without using `black`

Reply:
No, it's probably worth it for the consistency and linting.

Reply:
ok - here's https://github.com/linux-system-roles/storage/pull/205 for comparison - pretty much everything except the `black` formatting